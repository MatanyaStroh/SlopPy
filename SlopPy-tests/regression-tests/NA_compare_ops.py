# do a bunch of compare operations on an NA object

x = 1 / 0
assert type(x) is NA

assert not (x < 5)
assert not (x <= 5)
assert not (x > 5)
assert not (x >= 5)
assert not (x == 5)
assert not (x != 5)
