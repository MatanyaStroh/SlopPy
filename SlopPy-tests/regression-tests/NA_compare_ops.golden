{NA <type 'exceptions.ZeroDivisionError'> 'integer division or modulo by zero'
 Traceback (most recent call last):
  File "regression-tests/NA_compare_ops.py", line 3, in <module>
    x = 1 / 0
}
{PyObject_RichCompare(NA,*)=False | regression-tests/NA_compare_ops.py <module>:6}
{PyObject_RichCompare(NA,*)=False | regression-tests/NA_compare_ops.py <module>:7}
{PyObject_RichCompare(NA,*)=False | regression-tests/NA_compare_ops.py <module>:8}
{PyObject_RichCompare(NA,*)=False | regression-tests/NA_compare_ops.py <module>:9}
{PyObject_RichCompare(NA,*)=False | regression-tests/NA_compare_ops.py <module>:10}
{PyObject_RichCompare(NA,*)=False | regression-tests/NA_compare_ops.py <module>:11}
